### PR TITLE
New version: OpenShot.OpenShot version 3.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
### Result: Installation Failed
|             | Manifest                 | Add/Remove Programs                   |
| ----------- | ------------------------ | ------------------------------------- |
| Name        | OpenShot Video Editor |     |
| Version     | 3.0.0     |  |
| Publisher   | OpenShot Studios, LLC   |       |
| ProductCode |           |     |
###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/91034)

### CODE REVIEW

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:22]:
```suggestion
- Architecture: x64
```

Comment (stephengillie) [reply manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:22]: From the validation error, I'm guessing that the x86 version actually needs a 64-bit OS. 
image

**Review by stephengillie — DISMISSED** [review]

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+- Architecture: x64
```
Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:22]:
```suggestion
```

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86_64.exe
```
Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:23]:
```suggestion
```

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  InstallerSha256: 39B5AC64311FE5DE96CDF1C6660BD6A35EF1B1213EE48047E48932CABA2E0E91
```
Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:24]:
```suggestion
```

Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:24]:
```suggestion
```

Comment (stephengillie) [reply manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:24]: Removing the x86 installer, as it appears to require x64 architecture, and having two x64 entries is causing the manifest to fail validation.

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+- silentWithProgress
+UpgradeBehavior: install
+FileExtensions:
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+ManifestType: installer
```
Comment (stephengillie) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:22]:
```suggestion
- Architecture: x64
  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86_64.exe
  InstallerSha256: 39B5AC64311FE5DE96CDF1C6660BD6A35EF1B1213EE48047E48932CABA2E0E91
 ManifestType: installer
```

File: manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+- .osp
+ReleaseDate: 2022-12-08
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86.exe
+  InstallerSha256: ED9FEFD35989B512E962DB7207187B3AE4E94CB2DCD4C233443C03BA180C86B6
+- Architecture: x64
+  InstallerUrl: https://github.com/OpenShot/openshot-qt/releases/download/v3.0.0/OpenShot-v3.0.0-x86_64.exe
+  InstallerSha256: 39B5AC64311FE5DE96CDF1C6660BD6A35EF1B1213EE48047E48932CABA2E0E91
+ ManifestType: installer
```
Comment (mdanish-kh) [inline manifests/o/OpenShot/OpenShot/3.0.0/OpenShot.OpenShot.installer.yaml:25]:
```suggestion
ManifestType: installer
```
Fix indent

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**